Sophia Sun Project Pitch

```diff
@@ -0,0 +1,244 @@
+
+I will have some sort of array that contains all the possible notes in a chord. The array starts with A, and each note is a half step apart: 
+A A# Ab B B# Bb C C# Cb D D# Db E Eb F F# G G# Gb 
+Index 0: A
+Index 1: A# / Bb
+Index 2: B 
+Index 3: B# / C 
+Index 4: C# / Db
+Index 5: D 
+Index 6: D# / Eb
+Index 7: E
+Index 8: F
+Index 9: F# / Gb
+Index 10: G
+Index 11: G# / Ab
+
+My program takes the root of the chord and the quality of the chord and finds the notes that are in this chord. If the input specifies the root but not the quality of the chord, then I know that a lower case root represents a minor chord with that root, and an upper case root represents a major chord with that root. For example, that means that the input string BMin is the same chord as b. 
+
+All chords of the same quality (Major, minor, diminished, augmented, diminished 7th, Dominant 7th, Major 7th, etc) are composed of notes that are the same interval apart. For example, an A major chord is A C# E, and a C major chord is C E G. As you can see from the array above, 
+A = index 0, 
+C# = index 4, 
+E = index 7, 
+and 
+C = index 3,
+E = index 7, 
+G = index 10. 
+
+As you can tell, the difference between the first note and the second note is 7-3 = 4-0 = 4 half steps.
+The difference between the second note and the third note is 10-7 = 7-4 = 3 half steps. I will tell my program what set of intervals are between the different notes depending on the quality of the chord, and it will return the notes that are in the chord!!
+Then, the program would look at what inversion was specified (and if there was no inversion, assume a root position of the chord), and rearrange the order of the notes in the chord accordingly.
+
+Then the program compiles my DSL into a LilyPond program/code, 
+and LilyPond actually creates the sheet music for the user based on the notes and the order of the notes generated by my 
+program. LilyPond then does all the hard work of displaying the notes.
+
+# Is there a computational model that corresponds with your domain? 
+
+I'm not sure what the computational model means, but if it's what I understand, 
+I think I already described it above when I explained how I train the computer to output notes based on root, chord quality, and inversion.
+
+# How does a program interact with the user? 
+
+The program asks the user to submit a txt file with the list of chords, each chord on one line of the txt file. 
+
+# What kinds of semantic (i.e., non-syntax) errors might occur, and how might they be communicated to the user?_
+
+There are ways to express a chord that may be outside of what I'm programming my parser to be able to understand. For example,
```

Comment:
This sounds it would be a common syntax error, since users would be scraping these chords from some website. I don't think it's unreasonable to design the syntax with some restrictions in mind, as opposed to trying to parse every possible input format. For example, requiring minor chords to be lowercase instead of denoted with `Min`. I don't think this requires a lot of domain expertise from the user, provided that the program gives helpful error messages.